src/server.ts

```diff
@@ -154,7 +211,12 @@ export class RecordReplayServer {
         this.unloadTape();
         res.end(`Unloaded tape`);
       }
+      return;
     }
+
+    // If we got here, we don't know what to do. Return a 404.
+    res.statusCode = 404;
+    res.end(`Unhandled proxay request.\n\n${JSON.stringify(request)}`);
```

Comment:
This is also a bug fix. Currently if a `__proxay/blah` request is made for an unhandled route, the server hangs indefinitely. This adds a 404 response and logs out the request for debugging.